app/views/assignment/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Kitchen Schedule</h1>
+
+<table>
+
+<% @weeks.each do |w| %>
+  <tr>
+    <td>
+      <%= "#{w.starts_at.strftime('%B %d, %Y')}" %>
+    </td>
+    <% @assignments.where(week: w).each do |a| %>
```

Comment:
Expand the single letter variables.

This line should be `week.employees.each do |employee|`